test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -27,12 +28,17 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('visual builder', function describeIndexTests() {
     this.tags('includeFirefox');
+
+    before(async () => {
+      await PageObjects.visualize.initTests();
+    });
+
     beforeEach(async () => {
-      await security.testUser.setRoles([
-        'kibana_admin',
-        'test_logstash_reader',
-        'kibana_sample_admin',
-      ]);
+      await security.testUser.setRoles(
+        ['kibana_admin', 'test_logstash_reader', 'kibana_sample_admin'],
+        false
+      );
+      await initTests();
```

Comment:
This still seems wrong.  in the before method on line 33 we're calling `await PageObjects.visualize.initTests();` but then still calling a local `initTests()` which I think is the one in the visualize_page?

Reply:
Oh darn, good catch!